app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -491,6 +492,24 @@ def test_results_ready_false(self):
         )
         self.assertFalse(experiment.results_ready)
 
+    @parameterized.expand([(settings.DEV_USER_EMAIL, True), ("jdoe@mozilla.org", False)])
+    @mock.patch("django.conf.settings.SKIP_REVIEW_ACCESS_CONTROL_FOR_DEV_USER", True)
```

Comment:
Oh you don't have to use mock for this django has a dealy for it

﻿﻿https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.override_settings

Reply:
Oh.. 🤦 I've totally used that before too